applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -78,6 +94,17 @@ public static ConfigurationAsyncClientBuilder builder() {
      * Adds a configuration value in the service if that key and label does not exist. The label value of the
      * ConfigurationSetting is optional.
      *
+     * <p><strong>Code Samples</strong></p>
+     * <pre>
```

Comment:
Newline before the `pre` - try to enforce consistency on yourself in every aspect! :-)